packages/tpetra/core/src/Tpetra_CrsMatrix_def.hpp

```diff
@@ -5442,7 +5449,14 @@ namespace Tpetra {
          std::runtime_error, "X and Y may not alias one another.");
     }
 
-    lclMatrix_->apply (X_lcl, Y_lcl, mode, alpha, beta);
+    LocalOrdinal nrows = getNodeNumRows();
+    LocalOrdinal maxRowImbalance = 0;
+    if(nrows != 0)
+      maxRowImbalance = getNodeMaxNumRowEntries() - (getNodeNumEntries() / getNodeNumRows());
```

Comment:
Reuse nrows instead of getNodeNumRows()?

Reply:
Good idea